kotlinlib/src/mill/kotlinlib/KotlinMavenModule.scala

```diff
@@ -0,0 +1,29 @@
+package mill.kotlinlib
+
+import mill.T
+
+/**
+ * A [[KotlinModule]] with a Maven compatible directory layout.
+ */
+trait KotlinMavenModule extends KotlinModule { outer =>
```

Comment:
Can we inherit from `mill.javalib.MavenModule` here? It should look more or less identical to `SbtModule`